### PR TITLE
Restructure docs

### PR DESCRIPTION
**Pull request by AaronCrawfis**
This PR restructures the docs based on the new diataxis framework.

Content updates were minimal, only changing page content to make it fit the new framework.

There are many guides to be written so spots in the docs might be a little sparse, but we'll fill them in.

### CODE REVIEW

File: docs/content/author-apps/howto-vscode/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
```
Comment (Reshrahim) [inline docs/content/author-apps/howto-vscode/index.md]: Can we move this page to a higher level section on "installation"

File: docs/content/deploy-apps/howto-rad-cli/index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+---
```
Comment (Reshrahim) [inline docs/content/deploy-apps/howto-rad-cli/index.md]: This also makes a good page for top level "installation" section

File: docs/content/getting-started/index.md [hunk 3]
```diff
@@ -126,7 +126,7 @@ First add some new code to `app.bicep` by pasting in the content below at the en
 
 {{< rad file="snippets/app-with-redis-snippets.bicep" embed=true marker="//REDIS" markdownConfig="{linenos=table,linenostart=21}" >}}
 
-Next, update your container definition to include `connections` inside `properties`. This code creates a connection between the container and the database. Based on this connection, Radius will [inject environment variables]({{< ref "container#connections" >}}) into the container that inform the container how to connect. You will view these in the next step.
+Next, update your container definition to include `connections` inside `properties`. This code creates a connection between the container and the database. Based on this connection, Radius will [inject environment variables]({{< ref "/guides/author-apps/containers#connections" >}}) into the container that inform the container how to connect. You will view these in the next step.
```
Comment (willtsai) [inline docs/content/getting-started/index.md]: nitpick for first app experience:

```suggestion
Next, update your container definition to include `connections` inside `properties`. This code creates a connection between the container and the database. Based on this connection, Radius will [inject environment variables]({{< ref "/guides/author-apps/containers/overview#connections" >}}) into the container that inform the container how to connect. You will view these in the next step.
```

**Review by willtsai — CHANGES_REQUESTED** [review]
One minor change suggested. Reviewed "First app" in detail and the other pages/menu bar at a higher level. New structure looks great and provides a great framework for us to continue building upon!

File: docs/content/community/contributing/contributing-docs/index.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 type: docs
-title: "Contributing to Radius documentation"
-linkTitle: "Docs"
+title: "How-To: Contribute to the Radius documentation"
+linkTitle: "How-To: Contribute docs"
```
Comment (Reshrahim) [inline docs/content/community/contributing/contributing-docs/index.md]:
```suggestion
linkTitle: "Contribute to docs"
```

File: docs/content/community/maintainers/docs-maintainers.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 type: docs
-title: "Maintaining Radius docs"
-linkTitle: "Docs"
+title: "How-To: Maintain the Radius docs"
+linkTitle: "How-To: Maintain docs"
```
Comment (Reshrahim) [inline docs/content/community/maintainers/docs-maintainers.md]:
```suggestion
linkTitle: "Maintain docs"
```

File: docs/content/guides/author-apps/aws/howto-aws-resources/index.md [hunk 6]
```diff
@@ -1,14 +1,14 @@
 ---
 type: docs
-title: "Quickstart: Deploy AWS resources"
-linkTitle: "Deploy AWS resources"
+title: "How-To: Deploy AWS resources"
+linkTitle: "How-To: Use AWS resources"
```
Comment (Reshrahim) [inline docs/content/guides/author-apps/aws/howto-aws-resources/index.md]:
```suggestion
linkTitle: "Deploy AWS resources"
```

File: docs/content/guides/author-apps/azure/howto-azure-resources/index.md [hunk 7]
```diff
@@ -1,15 +1,15 @@
 ---
 type: docs
-title: "Quickstart: Connect a container to an Azure resource"
-linkTitle: "Connect to Azure resources"
+title: "How-To: Connect a container to an Azure resource"
+linkTitle: "How-To: Use Azure resources"
```
Comment (Reshrahim) [inline docs/content/guides/author-apps/azure/howto-azure-resources/index.md]:
```suggestion
linkTitle: "Deploy Azure resources"
```

File: docs/content/guides/author-apps/containers/howto-environment-variables/index.md [hunk 8]
```diff
@@ -1,27 +1,27 @@
 ---
 type: docs
-title: "Quickstart: Environment Variables"
-linkTitle: "Environment variables"
+title: "How-To: Set environment variables on a container"
+linkTitle: "How-To: Env vars"
```
Comment (Reshrahim) [inline docs/content/guides/author-apps/containers/howto-environment-variables/index.md]:
```suggestion
linkTitle: "Set Env vars"
```

File: docs/content/guides/author-apps/containers/howto-keyvault-volume/index.md [hunk 9]
```diff
@@ -1,15 +1,15 @@
 ---
 type: docs
-title: "Quickstart: Mount an Azure Key Vault as a volume to a container"
-linkTitle: "Persistent Key Vault volume"
+title: "How-To: Mount an Azure Key Vault as a volume to a container"
+linkTitle: "How-To: Mount a Key Vault"
```
Comment (Reshrahim) [inline docs/content/guides/author-apps/containers/howto-keyvault-volume/index.md]:
```suggestion
linkTitle: "Mount a Key Vault"
```

File: docs/content/guides/author-apps/containers/howto-volumes/index.md [hunk 10]
```diff
@@ -1,26 +1,26 @@
 ---
 type: docs
-title: "Quickstart: Mount a volume to a container"
-linkTitle: "Volume mounts"
+title: "How-To: Mount a volume to a container"
+linkTitle: "How-To: Volumes"
```
Comment (Reshrahim) [inline docs/content/guides/author-apps/containers/howto-volumes/index.md]:
```suggestion
linkTitle: "Mount a volume"
```

File: docs/content/guides/author-apps/howto-vscode/index.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
+---
+type: docs
+title: "How-To: Install the Radius-Bicep VSCode extension"
+linkTitle: "How-To: VS Code"
```
Comment (Reshrahim) [inline docs/content/guides/author-apps/howto-vscode/index.md]:
```suggestion
linkTitle: "Install VS Code extension"
```

File: docs/content/guides/deploy-apps/howto-cicd/index.md [hunk 12]
```diff
@@ -1,10 +1,10 @@
 ---
 type: docs
-title: "Quickstart: Github Actions CI/CD integration"
-linkTitle: "GitHub Actions CI/CD"
-description: "Learn about adding your Radius apps to your deployment pipelines" 
-weight: 300
-categories: "Quickstart"
+title: "How-To: Deploy an application with Github Actions"
+linkTitle: "How-To: GitHub Actions"
```
Comment (Reshrahim) [inline docs/content/guides/deploy-apps/howto-cicd/index.md]:
```suggestion
linkTitle: "Deploy with GitHub Actions"
```

File: docs/content/guides/deploy-apps/howto-delete/index.md [hunk 13]
```diff
@@ -1,9 +1,9 @@
 ---
 type: docs
 title: "How-To: Delete an application from a Radius environment"
-linkTitle: "Delete applications"
+linkTitle: "How-To: Delete apps"
```
Comment (Reshrahim) [inline docs/content/guides/deploy-apps/howto-delete/index.md]:
```suggestion
linkTitle: "Delete apps"
```

File: docs/content/guides/deploy-apps/howto-deploy/index.md [hunk 14]
```diff
@@ -1,9 +1,9 @@
 ---
 type: docs
 title: "Overview: Deploying applications into a Radius environment"
-linkTitle: "Deploy applications"
+linkTitle: "How-To: Deploy apps"
```
Comment (Reshrahim) [inline docs/content/guides/deploy-apps/howto-deploy/index.md]:
```suggestion
linkTitle: "Deploy apps"
```

File: docs/content/guides/deploy-apps/howto-rad-cli/index.md [hunk 15]
```diff
@@ -0,0 +1,95 @@
+---
+type: docs
+title: "How-To: Install the rad CLI"
+linkTitle: "How-To: rad CLI"
```
Comment (Reshrahim) [inline docs/content/guides/deploy-apps/howto-rad-cli/index.md]:
```suggestion
linkTitle: "Install rad CLI"
```

File: docs/content/guides/operations/kubernetes/_index.md [hunk 16]
```diff
@@ -0,0 +1,9 @@
+---
+type: docs
+title: "Kubernetes platform"
+linkTitle: "Kubernetes"
+description: "Learn how Radius can run on Kubernetes"
+weight: 10
+categories: "How-To"
```
Comment (Reshrahim) [inline docs/content/guides/operations/kubernetes/_index.md]:
```suggestion
```

File: docs/content/guides/operations/providers/howto-cloud-providers/index.md [hunk 17]
```diff
@@ -1,26 +1,13 @@
 ---
 type: docs
 title: "How-To: Setup Cloud providers"
-linkTitle: "Cloud providers"
+linkTitle: "How-To: Setup providers"
```
Comment (Reshrahim) [inline docs/content/guides/operations/providers/howto-cloud-providers/index.md]:
```suggestion
linkTitle: "Setup Cloud providers"
```

File: docs/content/guides/operations/workspaces/howto-workspaces/index.md [hunk 18]
```diff
@@ -1,100 +1,12 @@
 ---
 type: docs
-title: "Overview: Radius workspaces"
-linkTitle: "Workspaces"
+title: "How-To: Use Radius workspaces"
+linkTitle: "How-To: Workspaces"
```
Comment (Reshrahim) [inline docs/content/guides/operations/workspaces/howto-workspaces/index.md]:
```suggestion
linkTitle: "Use Workspaces"
```

File: docs/content/guides/recipes/howto-author-recipes/index.md [hunk 19]
```diff
@@ -1,7 +1,7 @@
 ---
 type: docs
-title: "Author Custom Radius Recipes"
-linkTitle: "Custom Recipes"
+title: "How-To: Author Custom Radius Recipes"
+linkTitle: "How-To: Author Recipes"
```
Comment (Reshrahim) [inline docs/content/guides/recipes/howto-author-recipes/index.md]:
```suggestion
linkTitle: "Author Recipes"
```

**Review by Reshrahim — CHANGES_REQUESTED** [review]
Overall the new structure is great!!. Suggested some changes on removing "how-to" in the link title and making the title task oriented.

File: docs/content/guides/author-apps/application/overview/index.md [hunk 20]
```diff
@@ -1,8 +1,8 @@
 ---
 type: docs
-title: "Radius application"
-linkTitle: "Application"
-description: "Learn how to model your application and its services in Radius"
+title: "Overview: Radius applications"
```
Comment (jasonviviano) [inline docs/content/guides/author-apps/application/overview/index.md:3]: I think for the `title` leaving the name for all the sections that are accessed through diving into specific topics should probably stay as just "Overview" versus "Overview: Topic" makes more sense as the user should know by the time they click on Applications that the Overview page found here is for Applications.

File: docs/content/guides/author-apps/howto-vscode/index.md [hunk 21]
```diff
@@ -0,0 +1,71 @@
...
+   {{< /tabs >}}
+
+2. Install the `.vsix` file:
+
+   {{< tabs UI Terminal >}}
+
+   {{% codetab %}}
+   In VSCode, manually install the extension using the *Install from VSIX* command in the Extensions view command drop-down.
+
+   <img src="./vsix-install.png" alt="Screenshot of installing a vsix extension" width=400>
```
Comment (Reshrahim) [inline docs/content/guides/author-apps/howto-vscode/index.md:50]: Images are not rendering 
image

File: docs/content/guides/author-apps/howto-vscode/index.md [hunk 22]
```diff
@@ -0,0 +1,71 @@
+---
+type: docs
+title: "How-To: Install the Radius-Bicep VSCode extension"
```
Comment (Reshrahim) [inline docs/content/guides/author-apps/howto-vscode/index.md:3]: Should we move this content inside Installation page? This feels out of place under authoring

**Review by Reshrahim — DISMISSED** [review]

**Review by willtsai — DISMISSED** [review]

**Review by willtsai — APPROVED** [review]
Approving the overall structure changes, we can open additional PRs to make changes to titles, ordering, etc.

File: docs/content/guides/author-apps/howto-vscode/index.md (hunk 21 above)
Comment (AaronCrawfis) [reply docs/content/guides/author-apps/howto-vscode/index.md:50]: Fixed

File: docs/content/guides/author-apps/howto-vscode/index.md (hunk 22 above)
Comment (AaronCrawfis) [reply docs/content/guides/author-apps/howto-vscode/index.md:3]: Discussed in PM meeting, will pickup in PR to templatize this information and place it in both places

**Pull request merged**